backend/icpy/agent/core/llm/anthropic_client.py

```
@@ -0,0 +1,24 @@
+from .base import BaseLLMClient, ProviderNotConfigured
+from ...helpers import OpenAIStreamingHandler
+from ...clients import get_anthropic_client
+
+
+class AnthropicClientAdapter(BaseLLMClient):
+    def stream_chat(
+        self,
+        *,
+        model: str,
+        messages: List[Dict[str, Any]],
+        tools: Optional[List[Dict[str, Any]]] = None,
```

Ruff (0.14.3)

Unused method argument: `tools`

(ARG002)